jira-macro/jira-macro-charts/src/main/java/org/xwiki/contrib/jira/charts/bidimensionalgrid/JIRAField.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.contrib.jira.charts.bidimensionalgrid;
+
+/**
+ * Representation of possible fields to select for x and y axises fields in bidimensional table charts.
+ *
+ * @version $Id$
+ * @since 10.0
+ */
+// TODO: this should probably be improved to not rely on an enum, and to allow giving custom fields...
+// see also JIRAField defined in jira macro
+public enum JIRAField
```

Comment:
Just to mention it, better check our rule regarding the use of the Unstable annotation on contrib extensions.
Given the TODO above, looks like something unstable.

Reply:
AFAIR we don't really use Unstable annotation in contrib extension since we don't have check to remove them, do we?

Reply:
I added them anyway to flag that I'm not yet entirely convinced about the whole API of macro parameters and that it might change.